src/backend/TrafficCourts/Workflow.Service/Consumers/EmailVerificationReceivedConsumer.cs

```diff
@@ -17,12 +17,14 @@ public class SetEmailVerifiedOnDisputeInDatabase : IConsumer<EmailVerificationSu
     private readonly ILogger<SetEmailVerifiedOnDisputeInDatabase> _logger;
     private readonly IOracleDataApiService _oracleDataApiService;
     private readonly IConfirmationEmailTemplate _confirmationEmailTemplate;
+    private readonly IDisputantEmailUpdateSuccessfulTemplate _emailUpdateSuccessfulTemplate;
 
-    public SetEmailVerifiedOnDisputeInDatabase(ILogger<SetEmailVerifiedOnDisputeInDatabase> logger, IOracleDataApiService oracleDataApiService, IConfirmationEmailTemplate confirmationEmailTemplate)
+    public SetEmailVerifiedOnDisputeInDatabase(ILogger<SetEmailVerifiedOnDisputeInDatabase> logger, IOracleDataApiService oracleDataApiService, IConfirmationEmailTemplate confirmationEmailTemplate, IDisputantEmailUpdateSuccessfulTemplate updateRequestReceivedTemplate)

```

Comment:
For this to work, don't you also need to register the template with EmailTemplateExtensions?

Reply:
Yes, that's right. Good catch! Forgot to do that.